mmv1/templates/terraform/iam/example_config_body/cloudfunctions2_function.tf.erb

```diff
@@ -0,0 +1,2 @@
+  cloud_function = google_cloudfunctions2_function.%{primary_resource_id}.name
```

Comment:
Why do we need this rather than having the location be pulled from the resource's ID automatically?

Reply:
I'm not sure, for some reason it was reading `us-central1-a` rather than `us-central1` (I assume from `GOOGLE_ZONE`). if you have thoughts on that, I'd be happy change it, but I wasn't able to find a different way around it.

Reply:
Huh... During the setup or during the import step? That's unusual, but I could see it happening

Reply:
During setup